app-backend/api/src/main/scala/project/ProjectLayerTaskRoutes.scala

```diff
@@ -99,6 +99,9 @@ trait ProjectLayerTaskRoutes
       }
   }
 
+  // TODO: TaskPropertiesCreate's last param needs to be a real annotate project's ID
+  // It is now just a regular project ID
+  // But this function will change after hooking tasks up with annotation projects
```

Comment:
I wonder if maybe a better solution here is to take all the foreign keys out of the `Create` case class and get them from routes. It's tough to see, but we [already](https://github.com/raster-foundry/raster-foundry/blob/6561c3924ca7c5748c300ce284fecbc74cf02e84/app-backend/api/src/main/scala/project/Routes.scala#L308) have the API route nesting that this would require (though obviously that will change soon -- but we should have the annotation project id instead), so it might be easier to assume that we're getting that information from somewhere other than the `Task.Create`. This is something to consider for when we get to remove the project layer and project IDs from tasks.